src/util/std_expr.h

```diff
@@ -4750,4 +4750,49 @@ inline void validate_expr(const popcount_exprt &value)
   validate_operands(value, 1, "popcount must have one operand");
 }
 
+/// this is a parametric version of an if-expression: it returns
+/// the value of the first case (using the ordering of the operands)
+/// whose condition evaluates to true.
```

Comment:
No else case? It's undefined if none of the conditions pass?

Reply:
Yes, this is bugging me as well.
The origin of this expression is SMV, where it is common to end with a 'true' case.
If you don't, you get non-determinism.